qutebrowser/keyinput/basekeyparser.py

```diff
@@ -315,6 +318,13 @@ def _handle_result(self, info: keyutils.KeyInfo, result: MatchResult) -> None:
             assert result.command is not None
             self._debug_log("Definitive match for '{}'.".format(
                 result.sequence))
+
+            max_digits = sys.get_int_max_str_digits()
+            if len(self._count) > max_digits:
```

Comment:
This doesn't exist with older Python patch releases, so you'd need to do something like:

```suggestion
            try:
                max_digits = sys.get_int_max_str_digits()
            except AttributeError:
                max_digits = None
            if max_digits is not None and len(self._count) > max_digits:
```

or so.